network-api/networkapi/utility/faker/streamfield_provider.py

```diff
@@ -212,6 +212,43 @@ def generate_full_width_image_field():
     })
 
 
+def generate_dear_internet_intro_text_field():
+    text = f'<p>{fake.paragraph(nb_sentences=3, variable_nb_sentences=True)}</p>'
+
+    return generate_field('intro_text', text)
+
+
+def generate_dear_internet_letter_field():
+    author_name = fake.name()
+    author_description = ''.join(
+        (
+            '<p>',
+            f'<a href="{fake.url(schemes=["https"])}" target="_blank">{author_name}</a>',
+            f' is {" ".join(fake.words(nb=15))}. {fake.sentence()}',
+            '</p>',
+        )
+    )
+
+    letter = f'<p>{fake.paragraph(nb_sentences=10, variable_nb_sentences=True)}</p>'
+
+    attributes = {
+        'author_description': author_description,
+        'author': author,
```

Comment:
should this be `author_name`? `author` no longer exists

Reply:
you are totally right! I replaced text on the wrong line 😂